### ISSUE TITLE
Chat – “Hmm…It’s not here” appears when open Distance request in RHP and navigate via link

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#33132]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.13-0
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

## Action Performed:
1. Go to https://staging.new.expensify.com/
2. Click on FAB
3. Select "Request Money"
4. Click on the "Distance" tab
5. Change the URL and navigate to https://staging.new.expensify.com/request/new/participants

## Expected Result:
Page redirected back to the "Distance" tab

## Actual Result:
“Hmm… It’s not here” appears for a moment when open Distance request in RHP and navigate via changed link https://staging.new.expensify.com/request/new/participants

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/e1e23c9f-a0ab-4df7-9f9b-04d16a43393e

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Julesssss** [comment]
Certainly not worth blocking the release. Removing the lable.

**Comment by paultsimura** [comment]
The old format links (e.g. https://staging.new.expensify.com/request/new/participants) are being deprecated with the refactored Request Money flow, aren't they?

cc: @tgolen

**Comment by tgolen** [comment]
Yeah, that's correct. I think this can just be closed since those old routes are intended not to work anymore.
